addons/context_actions/CfgContext.hpp

```diff
@@ -290,6 +290,71 @@ class EGVAR(context_menu,actions) {
             priority = 1;
         };
     };
+    class EditableObjects {
+        displayName = CSTRING(EditableObjects);
+        icon = QPATHTOEF(modules,ui\edit_obj_ca.paa);
+        priority = -98;
+        class Add {
+            displayName = ECSTRING(common,Add);
+            icon = QPATHTOF(ui\add_ca.paa);
+            priority = 2;
+            class 10m {
+                displayName = "10 m";
```

Comment:
The `m` should be localized for other languages.

Reply:
m is the SI symbol for meter, it is universal.

Reply:
There should be a space between the numerical value and the unit symbol.

Reply:
The `m` is an SI symbol but that doesn't make it fully universal. For example, Crylic would use м, not m. Granted, in this case, the difference is minor but something like kg it would be far more important (кг). It's good to start localizing them here and keep it simplified in the long run.